qiskit/visualization/state_visualization.py

```diff
@@ -1291,20 +1285,21 @@ def num_to_latex_ket(raw_value: complex, first_term: bool) -> Optional[str]:
         return None
 
 
-def numbers_to_latex_terms(numbers: List[complex]) -> List[str]:
+def numbers_to_latex_terms(numbers: List[complex], decimals: int = 10) -> List[str]:
```

Comment:
I was on the fence about asking for a release note for this new kwarg. I decided that we didn't need it because we don't re-export this function from the root of visualization or document it in the visualization api. But if people feel strongly that we should document this new kwarg we can always add it as part of the release prep.